src/Cassandra/ConsistencyLevel.cs

```diff
@@ -76,6 +76,19 @@ public enum ConsistencyLevel
         /// <summary>
         /// Similar to <c>One</c> but only within the DC the coordinator is in.
         /// </summary>
-        LocalOne = 0x000A
+        LocalOne = 0x000A,
+        YbConsistentPrefix = One,
+        YbStrong = Quorum,
```

Comment:
Nit: remove trailing comma.

Reply:
Trailing comma allows to add entries w/o modifying last line.
So it is better to review.